app/views/layouts/_navigation.html.erb

```diff
@@ -0,0 +1,18 @@
+<header class="container page-header">
+  <div>
+    <a href="" class="logo">LOGO</a>
+  </div>
+  <div class="tabs">
+    <a href="#" class="tab">TAB1</a>
+    <a href="#" class="tab">TAB2</a>
+    <a href="#" class="tab">TAB3</a>
+    <a href="#" class="tab">TAB4</a>
+  </div>
+  <div class="donate">
+    <button type="button" class="btn btn-success">
+      <a href="https://zerowastelviv.org.ua/pidtrymaty/" class="donate-btn"
+        >DONATE</a
+      >
```

Comment:
is there any reason of formatting like this?

Reply:
Thanks, done.